utils/grpc/runtime_test.go

```diff
@@ -0,0 +1,51 @@
+package grpc
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/tron-us/go-btfs-common/config"
+	"github.com/tron-us/go-btfs-common/protos/shared"
+	"github.com/tron-us/go-common/v2/constant"
+	"github.com/tron-us/go-common/v2/log"
+	"go.uber.org/zap"
```

Comment:
zap and log aren't needed, use only assert functions for unit tests

Reply:
okay.